### PR TITLE
Fix incorrect code generated for statics

### PR DESCRIPTION
**Pull request by meevee98**
**Summary or solution description**
The code generator was generating different static slots for the same static variable that is used in different files due to failing to get the internal unique id of this variable.

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-861mucfzd incorrect code generated for statics](https://app.clickup.com/t/861mucfzd)

File: boa3/internal/compiler/codegenerator/codegenerator.py [hunk 1]
```diff
@@ -399,41 +399,57 @@ def get_symbol(self, identifier: str, scope: Optional[ISymbol] = None, is_intern
...
+                            elif isinstance(attr, Package) and symbol_id in attr.inner_packages:
+                                found_id, found_symbol = symbol_id, attr.inner_packages[symbol_id]
+
+                        if found_id is None and is_internal:
+                            from boa3.internal.model import imports
+                            found_symbol = imports.builtin.get_internal_symbol(identifier)
+                            if isinstance(found_symbol, ISymbol):
+                                found_id = identifier
+
+        if found_id is not None:
```
Comment (ixje) [inline boa3/internal/compiler/codegenerator/codegenerator.py:444]:
```suggestion
        else:
```

**Review by ixje — APPROVED** [review]
tested on a couple of contracts and works as intended :+1:

File: boa3/internal/compiler/codegenerator/codegenerator.py (hunk 1 above)
Comment (ixje) [reply boa3/internal/compiler/codegenerator/codegenerator.py:444]: ah missed that! all good :)

**Pull request merged**